roadmap/implementers-guide/src/node/utility/provisioner.md

```diff
@@ -121,6 +173,26 @@ The subsystem should maintain a set of handles to Block Authorship Provisioning
 
 Forward the message to the appropriate Block Authorship Provisioning Job, or discard if no appropriate job is currently active.
 
-## Block Authorship Provisioning Job
+### Block Authorship Provisioning Job
```

Comment:
Can the whole jobs section be removed? We have an issue open to [remove references to jobs](https://github.com/paritytech/polkadot/issues/6526) but I think *some* subsystems still use this terminology.

Reply:
Seems reasonable to get rid of given the context you provided.